### ISSUE TITLE
Delete website files, when website is deleted from dashboard

### ISSUE THREAD

**Issue by leafes** [nostrband/nostr-site-manager#89]

**Comment by nostrband** [comment]
Can you please repeat the test? You've deleted at80.npub.pro and I see the code that deletes the files, and the site is no longer served. 

Looks like files aren't deleted only if deletion didn't go well.

**Comment by leafes** [comment]
@nostrband https://jam.dev/c/c12a1528-ef1f-4a62-be29-c09c3e4b2f42
yeah, i'm able to repeat that

**Comment by leafes** [comment]
It worked this way, website is actually deleted.

Do you think it’s necessary to unregister a service worker if a user visits a previously visited but now deleted website?

I’m thinking of a use case where, as a site author, I don’t want people with whom I’ve shared my site link before to be able to access my content through the link.

**Comment by nostrband** [comment]
> Do you think it’s necessary to unregister a service worker if a user visits a previously visited but now deleted website?

Yes I think renderer should notice the site deletion request and kill itself, maybe rename this issue to "Service worker should not render deleted sites"
